src/Adapter/Order/OrderPresenter.php

```diff
@@ -83,9 +83,11 @@ public function present($order)
             throw new \Exception('OrderPresenter can only present instance of Order');
         }
 
-        return array(
-            'products' => $this->getProducts($order),
-            'products_count' => count($this->getProducts($order)),
+        $products = $this->getProducts($order);
+
+        return [
+            'products' => $products,
+            'products_count' => count($products),
             'totals' => $this->getAmounts($order)['totals'],
             'subtotals' => $this->getAmounts($order)['subtotals'],
```

Comment:
Good idea, but can you please keep the long array syntax `array(...)`?

And it seems we can also store the result of `$this->getAmounts()`, in order to reuse it for `totals` and `subtotals`.

Reply:
Sure, ill update it.

But is there a particular reason why we have to use long array syntax?

Reply:
I'm suprised that long array syntax is needed because we can see short array syntax in core...

Reply:
Yeah, the coding rules are the same as PrestaShop 1.6, which is itself compatible with some PHP version unable to handle the short array syntax.